spec/features/user_signs_up_spec.rb

```diff
@@ -8,24 +8,31 @@
 
   # Acceptance Criteria:
   # * I must specify a valid email address,
-  #   password, and password confirmation
+  #   password, password confirmation, and personal information.
   # * If I don't specify the required information, I am presented with
-  #   an error message
+  #   an error message.
 
-  scenario 'provide valid registration information' do
-    visit new_user_registration_path
+  scenario 'User provides valid registration info' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.